the_encryptor_v2.py

```diff
@@ -1,8 +1,18 @@
+#!/usr/bin/env python
+
 import os
 from os.path import exists
 import time
 from tkinter import N
 from cryptography.fernet import Fernet as fn
+import sys
+import signal
+
+def exit_handler(sig, frame):
+    print("\n\nApplication exited succesfully\n\n")
+    sys.exit(0)
+
+signal.signal(signal.SIGINT, exit_handler)
```

Comment:
Yes, and also you can comment on the lines of code in question...

This bit of code handles when the system/user force quits the application e.g. `Ctrl+C`